mitodl/salt-ops
# Added OIDC settings for mitxonline

#### What are the relevant tickets?
Part of https://github.com/mitodl/mitxonline/issues/519

#### What's this PR do?

This adds some configuration settings recently introduced in MITx Online. They've been set manually on RC on a testing basis.

#### How should this be manually tested?
You can go through the steps in the updated documentation I have in a branch over there to be sure that both the documentation explains the setup well and that the code in this PR is valid: 

https://github.com/mitodl/mitxonline/blob/8cc703dbb9c7601ab7e67488abfb9676660f8036/docs/source/configuration/refine_admin.rst#deployments-rc-production-etc

Once that's configured, you can go login as a staff user and then go to `/staff-dashboard` on the app. You should be able to login and see a dashboard interface.
